### PR TITLE
fix: Add a sentence that clarify GDPR status

### LINKED ISSUE #829: Incomplete GDPR scope in the introduction
**What's the issue?**

[This section](https://github.com/OWASP/wstg/blob/master/document/2-Introduction/README.md#understand-the-scope-of-security)   describes the General Data Protection Regulation (GDPR) in an incomplete way : 

> For organizations based in EU countries, both country-specific regulation and EU Directives may apply. For example, Directive 
> 96/46/EC4 and Regulation (EU) 2016/679 (General Data Protection Regulation) make it mandatory to treat personal data in 
> applications with due care, whatever the application.

The GDPR target not only EU companies but businesses anywhere in the world as long as data from EU citizens is collected. 

A good source to check : https://gdpr.eu/what-is-gdpr

> The General Data Protection Regulation (GDPR) is the toughest privacy and security law in the world. Though it was drafted and 
> passed by the European Union (EU), it imposes obligations onto organizations anywhere, so long as they target or collect data 
> related to people in the EU.

**How do we solve it?**
Update the example.

Would you like to be assigned to this issue?
- [X] Assign me, please!

### PR DESCRIPTION
**Pull request by guilloux-valentin**
This PR closes #829 .

- [x] This PR handles the issue and requires no additional PRs.
- [x] You have validated the need for this change.

**What did this PR accomplish?**

- Add a sentence in the introduction that clarify GDPR status #829

### CODE REVIEW

File: document/2-Introduction/README.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ In modern development methodologies such as (but not limited to): agile, devops/
...
+It is important to know how much security a given project will require. The assets that are to be protected should be given a classification that states how they are to be handled (e.g., confidential, secret, top secret). Discussions should occur with legal council to ensure that any specific security requirements will be met. In the USA, requirements might come from federal regulations, such as the [Gramm-Leach-Bliley Act](https://www.ftc.gov/tips-advice/business-center/privacy-and-security/gramm-leach-bliley-act), or from state laws, such as the [California SB-1386](https://leginfo.legislature.ca.gov/faces/billTextClient.xhtml?bill_id=200120020SB1386). For organizations based in EU countries, both country-specific regulation and EU Directives may apply. For example, [Directive 96/46/EC4](https://ec.europa.eu/info/policies/justice-and-fundamental-rights_en) and [Regulation (EU) 2016/679 (General Data Protection Regulation)](https://gdpr-info.eu/) make it mandatory to treat personal data in applications with due care, whatever the application. Non-EU organisations, under certain circonstances, may also comply with the General Data Protection Regulation.
```
Comment (kingthorin) [inline document/2-Introduction/README.md:119]:
```suggestion
It is important to know how much security a given project will require. The assets that are to be protected should be given a classification that states how they are to be handled (e.g., confidential, secret, top secret). Discussions should occur with legal council to ensure that any specific security requirements will be met. In the USA, requirements might come from federal regulations, such as the [Gramm-Leach-Bliley Act](https://www.ftc.gov/tips-advice/business-center/privacy-and-security/gramm-leach-bliley-act), or from state laws, such as the [California SB-1386](https://leginfo.legislature.ca.gov/faces/billTextClient.xhtml?bill_id=200120020SB1386). For organizations based in EU countries, both country-specific regulation and EU Directives may apply. For example, [Directive 96/46/EC4](https://ec.europa.eu/info/policies/justice-and-fundamental-rights_en) and [Regulation (EU) 2016/679 (General Data Protection Regulation)](https://gdpr-info.eu/) make it mandatory to treat personal data in applications with due care, whatever the application. Non-EU organisations, under certain circonstances, may also be required to comply with the General Data Protection Regulation.
```

Dose that seem right?

Comment (guilloux-valentin) [reply document/2-Introduction/README.md:119]: Yes, it seems fine to me.

**Review by kingthorin — APPROVED** [review]
Thankd

**Comment by davidofell5** [comment]
This is david odell im not good at stuff if can call me so we can discuss
how to get my money ,6813553962.that my number this david odell 12,/09/1984
thank you.

**Pull request merged**